index.html

```diff
@@ -91,7 +91,7 @@ <h1>What to expect at HackHPI 2018</h1>
 					</p>
 				</div>
 				<div class="col-md-5">
-					<img src="../hackhpi_assets/intro-bg-2018.png" class="top50 img-responsive visible-md-block visible-lg-block">
+					<img src="./hackhpi_assets/intro-bg-2018.png" class="top50 img-responsive visible-md-block visible-lg-block">
```

Comment:
why did we change that anyways?

Reply:
We probably copied it from the `2017/index.html` where it would be correct.